packages/nuxt/src/module.ts

```diff
@@ -74,6 +74,15 @@ export default defineNuxtModule<ModuleOptions>({
 
     nuxt.hooks.hook('nitro:init', nitro => {
       if (serverConfigFile && serverConfigFile.includes('.server.config')) {
+        if (nitro.options.dev) {
+          consoleSandbox(() => {
+            // eslint-disable-next-line no-console
+            console.log(
+              '[Sentry] Your application is running in development mode. Note: @sentry/nuxt is in beta and may not work as expected on the server-side (Nitro).',
```

Comment:
I would mention more concretely what works and what doesn't so that users have a better understanding of what they should expect.